### DISCUSSION TITLE
Aerospace catch some window that dose not exist

### DISCUSSION THREAD

**Discussion by QuarterHeaven** [nikitabobko/AeroSpace#998] [potential-bugs]
I'm using Emacs with a plugin [welcome-dashboard](https://github.com/konrad1977/welcome-dashboard). After I launch Emacs, the window only have half screen. `aerospace list-windows --workspace visible` shows there are two windows with name 'Emacs', but only one has title. 

I tried to focus the window which doesn't have title, but it says can't find that window, while focusing the one which has title works fine. And that window doesn't exit even I closed Emacs.

<img width="381" alt="图片" src="https://github.com/nikitabobko/AeroSpace/assets/41909950/85159a7c-a2cb-4b49-a11c-b3fd82f15fe2">

<img width="392" alt="图片" src="https://github.com/nikitabobko/AeroSpace/assets/41909950/9e1d4cc4-2549-45a4-ad76-b20fd02949ae">

and because I can't focus the window, I can't use `aerospace close` to close it. So it will stay on my screen untill I reboot. Is there any way to solve this?

**Comment by mangas** [comment]
For me, moving `(tool-bar-mode -1)` to the window setup hook worked:

```
(add-hook 'window-setup-hook (lambda ()
			       (tool-bar-mode -1)
			       (inhibit-startup-message t)
			       (toggle-frame-maximized t)
			       ))
```

After this it stopped leaving phantom frames every start, hopefully this can help someone with the same issue.

**Comment by dhruvasagar** [comment]
@mangas That didn't work for me...
